content/learn/book/the-game-loop/time-and-timers.md

```diff
@@ -0,0 +1,277 @@
+
+[states]: ../control-flow/states.md
+[run conditions]: ../control-flow/run-conditions.md
+
+## Fixing your timestep
+
+Compensating for fluctating frame times using delta time is a great start.
+But as the timeless [fix your timestep article] explains for projects that require a higher level of stability and reproducibility,
+it's better to simply always advance time by a fixed amount.
+This is particularly important for physics and
+
+To understand how to work with fixed time in Bevy, we need to first learn a little bit about how [`Time`] actually works under the hood.
+As the excellent docs on [`Time`] explain, there's actually *three* distinct types of time being measured:
+
+- real time: the actual wall time
+  - use this for things like UI animations that you don't want to be affected by pausing
+- virtual time: the "in-game time"
+  - if you're using fixed time, this is useful for graphical effects
+  - otherwise, this is used for all of your gameplay logic as well
+- fixed time
+  - all of your gameplay logic should go here if you're using a fixed timestep approach
+
+When thinking about fixed time, it's important to clearly distinguish **frames** and **ticks**.
+A "frame" is one pass of the [`Main`] schedule, and corresponds to a single rendered frames.
+A "tick" is one pass of the [`FixedMain`] schedule, and corresponds to one iteration of gameplay logic.
+The ratio between frames elapsed and ticks elapsed is not constant: they may be faster or slower, depending
+on the rendering performance and your game's simulation needs.
+
+{% callout(type="info") %}
+Simply requesting [`Time`] in your systems will get you the correct flavor 90% of the time: virtual time in the [`Main`] schedule,
+and fixed time in the [`FixedMain`] schedule.
+To request a specific variation, change the implicit generic in [`Time`] from `()`
+to [`Real`], [`Virtual`] or [`Fixed`].
+{% end %}
+
+Now that we have the required vocabulary, let's go over exactly how the fixed timestep logic works in Bevy:
+
+- each frame, during the [`RunFixedMainLoop`] schedule, we determine how many times we should run the [`FixedMain`] schedule
+  - we add the the elapsed virtual time for the frame to a running time buffer
+  - if that time buffer is greater than or equal to [`Time<Fixed>::timestep`], we run the [`FixedMain`] schedule once, and subtract that timestep from our buffer
+- when the [`FixedMain`] schedule is evaluated, the various subschedules are iterated through, running [`FixedPreUpdate`], [`FixedUpdate`] and so on in order
+- within your fixed time systems, use the exact same delta time techniques to properly account for changes in your desired timestep during development
+
+This means that we may have 0, 1, 2 or more ticks per frame,
+with our fixed timestep logic running repeatedly until it's caught back up.
+For even more detail, check out the excellent documentation on [`Fixed`].
```

Comment:
Same as what Jan said earlier, Bevy referring to its own documentation as "excellent" in its own book feels quite weird. Something like "extensive" or "comprehensive" would be more fitting, or we could even just drop the adjective altogether.

Reply:
```suggestion
For even more detail, check out the documentation on [`Fixed`].
```